release_dashboard/lib/services/conductor.dart

```diff
@@ -17,6 +17,16 @@ abstract class ConductorService {
   /// Returns null when there is no active release, such as in the case when first initialized.
   pb.ConductorState? get state;
 
+  /// Returns the absolute path where the engine checkout is being saved.
+  ///
+  /// Can only be called after [createRelease] has been ran correctly.
+  get engineCheckoutDirectory;
```

Comment:
This should include the type to be returned.

https://dart-lang.github.io/linter/lints/always_declare_return_types.html

Reply:
oops, forgot about it. My bad.